### ISSUE TITLE
[Error] Missing measurement: tekton_tekton_chains_controller_workqueue_depth 

### ISSUE THREAD

**Issue by rh-rahulshetty** [openshift-pipelines/performance#17]
PromQL Query: `sum(watcher_workqueue_depth{container='tekton-chains-controller'})`

```
2024-03-22 14:56:27,670 root MainThread DEBUG /Getting data for measurements.tekton_tekton_chains_controller_workqueue_depth using Prometheus query sum(watcher_workqueue_depth{container='tekton-chains-controller'}) and step 15
2024-03-22 14:56:27,961 root MainThread DEBUG Response: {'status': 'success', 'data': {'resultType': 'matrix', 'result': []}}
2024-03-22 14:56:27,961 root MainThread ERROR Failed to measure measurements.tekton_tekton_chains_controller_workqueue_depth: missing 'response' in response's 'data'
Traceback (most recent call last):
  File "/tmp/tmp.bx7cCgx7gB/venv/src/opl-rhcloud-perf-team-core/core/opl/cluster_read.py", line 509, in __next__
    output = instance.measure(self, **self.config[i])
  File "/tmp/tmp.bx7cCgx7gB/venv/src/opl-rhcloud-perf-team-core/core/opl/cluster_read.py", line 135, in measure
    ), "missing 'response' in response's 'data'"
AssertionError: missing 'response' in response's 'data'
```

**Comment by rh-rahulshetty** [comment]
After debugging between different runs, it seems that the metric is missing while running the tests on OpenShift Pipeline 1.13. The same metric is available in Pipelines 1.14.

**Comment by rh-rahulshetty** [comment]
Closing this as it is not an issue.
